### PR TITLE
adding a few thoughts & a toggle clangen font setting

### PR DESCRIPTION
**Pull request by sarhecker**
minor additions to leader thoughts, a few more personality based text

### CODE REVIEW

**Comment by keyraven** [comment]
The clan-gen font toggles does not appear to work.  Is it still a work in progress?

**Comment by sarhecker** [comment]
> The clan-gen font toggles does not appear to work.  Is it still a work in progress? 

If you click the custom font button, the the app and reopen it it should work :^)

**Comment by keyraven** [comment]
> > The clan-gen font toggles does not appear to work.  Is it still a work in progress?
> 
> If you click the custom font button, the the app and reopen it it should work :^)

Ahhh, I see.  I tried to restart the game, but the old font is still there - I don't see the custom clangen font.

File: ClanGenRun.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
```
Comment (keyraven) [inline ClanGenRun.py:1]: Why has main.py been cleared? It seems all it's contents has been moved to this new ClanGenRun.py file. Was there a pressing need to do this?

**Comment by sarhecker** [comment]
> > > The clan-gen font toggles does not appear to work.  Is it still a work in progress?
> 
> > 
> 
> > If you click the custom font button, the the app and reopen it it should work :^)
> 
> 
> 
> Ahhh, I see.  I tried to restart the game, but the old font is still there - I don't see the custom clangen font. 

That's so bizarre it works fine for me...does the font name in the .json files change when you click the button?

File: scripts/game_structure/game_essentials.py [hunk 2]
```diff
@@ -660,13 +660,19 @@ def add_faded_offspring_to_faded_cat(self, parent, offspring):
...
         font_name='notosans',
         regular_path='resources/fonts/NotoSans-Medium.ttf',
         bold_path='resources/fonts/NotoSans-ExtraBold.ttf',
         italic_path='resources/fonts/NotoSans-MediumItalic.ttf',
         bold_italic_path='resources/fonts/NotoSans-ExtraBoldItalic.ttf'
     )
+    #clangen font path
+    manager.add_font_paths(
+        font_name='clangen',
+        regular_path='resources/fonts/clangen.ttf'
```
Comment (keyraven) [inline scripts/game_structure/game_essentials.py:674]: I see now. You successfully added a new font named "clangen" into the UIManager, all the text boxes have been themed to use "nonosans", by name.  To apply the custom font, you would need to load a different theme file for the new font, or load the clangen font with the font_name "notosans".  You would most likely also need to have bold and italic versions of the font, to make sure everything still works.

**Review by keyraven — CHANGES_REQUESTED** [review]

**Comment by keyraven** [comment]
> > > > The clan-gen font toggles does not appear to work.  Is it still a work in progress?
> > 
> > 
> > > 
> > 
> > 
> > > If you click the custom font button, the the app and reopen it it should work :^)
> > 
> > 
> > Ahhh, I see.  I tried to restart the game, but the old font is still there - I don't see the custom clangen font.
> 
> That's so bizarre it works fine for me...does the font name in the .json files change when you click the button?

Oh, are you changing the theme files with code?  I don't think that is a particularly good implementation.  Ideally, we should never be writing to the theme files. We are having issues with file nullification when we write to files. That's not too bad when it's the save-files, but if it clears the theme files, that would force re-install of the game.

**Comment by sarhecker** [comment]
> > > > > The clan-gen font toggles does not appear to work.  Is it still a work in progress?
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > If you click the custom font button, the the app and reopen it it should work :^)
> 
> > > 
> 
> > > 
> 
> > > Ahhh, I see.  I tried to restart the game, but the old font is still there - I don't see the custom clangen font.
> 
> > 
> 
> > That's so bizarre it works fine for me...does the font name in the .json files change when you click the button?
> 
> 
> 
> Oh, are you changing the theme files with code?  I don't think that is a particularly good implementation.  Ideally, we should never be writing to the theme files. We are having issues with file nullification when we write to files. That's not too bad when it's the save-files, but if it clears the theme files, that would force re-install of the game.   

Alrighty! I'll find a way to rework it but for now I'll close this pull request :P

**Pull request closed without merging**